packages/flow-core/src/components/f-accordion/f-accordion.ts

```diff
@@ -169,10 +189,10 @@ export class FAccordion extends FRoot {
 				<f-div
 					height="hug-content"
 					@click=${this.toggleAccordion}
-					align="middle-left"
+					align="top-left"
```

Comment:
This will break UI for people who already consumed, use part if this is for `f-document-viewer` use case